pkg/exec/exec.go

```diff
@@ -0,0 +1,40 @@
+package exec
```

Comment:
I figured I'd expand on this. In particular, I'd love to move towards breaking up our current utils package into the following set of packages:

- docker interactions
- files
- network
- user input / output (can probably be a single package)
- archive (zip, tar, etc)

None of this is urgent, of course, but I do feel we'd be better off if we had a set of smaller packages with defined concerns over one catchall util package.